### ISSUE TITLE
Should the detchannelmaps repo require PR approval for merges to develop?

### ISSUE THREAD

**Issue by bieryAtFnal** [DUNE-DAQ/detchannelmaps#45]
... during the normal part of the development cycle?

I created a version-bump PR, and I was a little surprised that it required approval from someone other than me.  Is this expected?  If so, please remind me of the reason for that.
Thanks!

**Comment by jcfreeman2** [comment]
No, requiring external approval for a PR to `develop` is obsolete. Having said that, the branch protection settings for this repo must not have been updated when we changed that policy a while back; I've updated the protection settings so PRs to develop won't require external approval.

**Comment by alessandrothea** [comment]
Actually, enforcing reviews on repositories with significant impact on the overall code like `*dataformats` and `detchannelmaps` could be beneficial.

**Comment by wesketchum** [comment]
Closing for now. Can reopen if we need to revisit PR approval rules, but it looks like it has develop protected, per usual rules.
